doc/advanced/02_starting_bft_blockchain.md

```diff
@@ -107,16 +107,16 @@ bft:
 
 ```yaml
 ---
-logger:
- verbosity: 4
- format: json
+log:
+  level: trace
+  format: json
 rest:
- listen: "127.0.0.1:8607"
+  listen: "127.0.0.1:8607"
 peer_2_peer:
- public_address: /ip4/127.0.0.1/tcp/8606
- topics_of_interests:
- messages: low
- blocks: normal
+  public_address: /ip4/127.0.0.1/tcp/8606
+  topics:
```

Comment:
I am not keen on changing this value. Here we are registering our level of interests into the given topics. Now it feels like the semantic is gone.